p:captcha callback support

gpoluch opened the pull request:
Fixes #1241 callback and callback-expired from reCaptcha v2 were not supported by p:captcha.
With this patch captcha provide two attributes for javascript callbacks:
- callback (executed when the user submits a successful captcha response)
- expired (executed when the recaptcha response expires and the user needs to solve a new captcha).

gpoluch committed:
p:captcha callback support
callback and callback-expired from reCaptcha v2 were not supported by p:captcha.
With this patch captcha provide two attributes for javascript callbacks:
- callback (executed when the user submits a successful captcha response)
- expired (executed when the recaptcha response expires and the user needs to solve a new captcha).

tandraschko commented:
I would apply your PR if it wouldn't destroy our formatting and i could see only the exact changes.

gpoluch committed:
Revert wrong formatted changes

gpoluch committed:
callback and callback-expired from reCaptcha v2 were not supported by p:captcha.
With this patch captcha provide two attributes for javascript callbacks:
- callback (executed when the user submits a successful captcha response)
- expired (executed when the recaptcha response expires and the user needs to solve a new captcha).

gpoluch commented:
Formatting corrected at 218fb2e.

tandraschko commented:
Merged - thanks for the PR!
